logging.rb

```diff
@@ -0,0 +1,25 @@
+require 'logger'
+
+module Logging
+  class << self
+    def logger
+      @logger ||= Logger.new STDOUT
+    end
+
+    def logger=(logger)
+      @logger = logger
+    end
+  end
+
+  def self.included(base)
+    class << base
+      def logger
+        Logging.logger
+      end
+    end
+  end
+
+  def logger
+    Logging.logger
+  end
+end
```

Comment:
@paulkerschen - that alternative doesn't allow me to insert logging statements into RSpec tests, which I need to be able to do.